### ISSUE TITLE
Meta message too annoying combined with signature tip

### ISSUE THREAD

**Issue by memeplex** [company-mode/company-mode#796]
I'm using elpy which provides call tooltips via eldoc. These tooltips already highlight the argument being typed. Now, the minibuffer is constantly going back and forth the signature tooltip and company's meta message, which doesn't add that much in this case, really.

I think this is related to https://github.com/company-mode/company-mode/issues/588.

It's really too annoying, I'm not fussing. Maybe it's better to tackle the issue at elpy side, I don't know.

**Comment by memeplex** [comment]
Could you suggest how to disable it or throttle it from elpy?

**Comment by dgutov** [comment]
But `meta` shows information about the symbol you're inserting. ElDoc doesn't really know about it, so it can't show the most pertinent tip.

So in this case I'd rather pick a solution that gives priority to Company over ElDoc.

**Comment by memeplex** [comment]
Right, it's better to leave this decision to the user, at  the moment I  still didn't understand quite well the frontend/backend stuff and how easy it is to disable one of them. I thought I'd already closed this one, sorry.

**Comment by dgutov** [comment]
It is indeed already closed. But the problem is unsolved.
